NimbleSurvey/Sources/Splash/SplashView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SplashView.swift
+//  NimbleSurvey
+//
+//  Created by Doan Thieu on 28/09/2022.
+//
+
+import SwiftUI
+
+struct SplashView: View {
+
+    @State private var showingLogo = false
+
+    var body: some View {
+        ZStack {
+            Image(Asset.Images.background.name)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .ignoresSafeArea()
+                .overlay(OverlayView())
+
+            Image(Asset.Images.logo.name)
+                .frame(maxWidth: .infinity)
+        }
+        .onAppear {
+            withAnimation(.easeInOut(duration: 1.0).delay(1.0)) {
+                showingLogo = true
+            }
+        }
+    }
+}
+
+struct SplashView_Previews: PreviewProvider {
```

Comment:
Please consider to follow our conventions

Reply:
Updated in [#a6e9d0f](https://github.com/Thieurom/ic-surveys-ios/pull/31/commits/a6e9d0feefbf7fa5aa9ddd166519970026f5215d)